Class/Tools.php

```diff
@@ -194,104 +256,155 @@ public static function wrap($text, $prefix, $postfix = '', $else = '')
         return $else;
     }
 
-    // return true if $n is among given parameters (more than one can be given)
+    /**
+     * Return true if $n is among given parameters (more than one can be given)
+     *
+     * @param mixed $n value tested
+     * @param mixed $m option(s)
```

Comment:
nechceš ten druhý a další parametr řešit jako v ifnull?

Reply:
Tady musí být zadány aspoň 2 parametry (jinak to nedává smysl).
A nebo nevím, jak si to řešení představuješ.

Reply:
Aha, musí být zadány alespoň dva. Ovšem $m stejně ve funkci není použit.

Reply:
Správně - není. Používám zde func_get_args(), ale současně, pokud bych někde napsal Tools::amon() jen s jedním parametrem, PHPčko mě upozorní, že musí být aspoň dva.